is_core/forms/fields.py

```diff
@@ -17,8 +17,8 @@ class ReadonlyField(forms.Field):
 
     def __init__(self, required=True, widget=None, label=None, initial=None,
                  help_text='', error_messages=None, show_hidden_initial=False,
-                 validators=[], localize=False):
-        super(ReadonlyField, self).__init__(required=False, widget=widget, label=label, initial=initial,
+                 validators=None, localize=False):
```

Comment:
proč se tady ty argumenty žerou a nepoužívaj? validators se zahodí, localize se zahodí, atd.

Reply:
protože je tam nechceme, je to readonly žádné validace tam nejsou